gammacat/webpage.py

```diff
@@ -44,8 +45,26 @@ def run(self):
         log.info('Make webpage ...')
         self.make_source_list_page()
         self.make_source_detail_pages()
+        self.make_publication_list()
         self.copy_data()
 
+    def make_publication_list(self):
+        publications = []
+        for x in (Path(gammacat_info.out_path / 'data')).iterdir():
```

Comment:
It is much more easier to get all references by scanning through the output/data folder instead of getting them by scanning through the output-index file. 
Firstly the entries in the index file are sorted by source_id and secondly the references can be doubled because there are more than one dataset for one reference.

Reply:
Please don't do that. The whole point of the index files it to avoid scrambling through directories from all over the place to see what's there. I think with the resource index helper class in Gammapy it's already possible to do anything you need here with one or two lines, i.e. select subsets of data as you like.